apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/utils/RDATestUtils.java

```diff
@@ -0,0 +1,191 @@
+package gov.cms.bfd.server.war.utils;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Sets;
+import gov.cms.bfd.model.rda.PreAdjFissClaim;
+import gov.cms.bfd.model.rda.PreAdjFissProcCode;
+import gov.cms.bfd.model.rda.PreAdjMcsClaim;
+import gov.cms.bfd.model.rda.PreAdjMcsDetail;
+import gov.cms.bfd.model.rda.PreAdjMcsDiagnosisCode;
+import gov.cms.bfd.model.rif.schema.DatabaseSchemaManager;
+import gov.cms.bfd.model.rif.schema.DatabaseTestUtils;
+import java.math.BigDecimal;
+import java.time.Instant;
+import java.time.LocalDate;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Consumer;
+import javax.persistence.EntityManager;
+import javax.persistence.Persistence;
+import javax.sql.DataSource;
+
+public class RDATestUtils {
+
+  public static final String PERSISTENCE_UNIT_NAME = "gov.cms.bfd.rda";
+
+  private EntityManager entityManager;
+
+  public void init() {
+    final DataSource dataSource = DatabaseTestUtils.get().getUnpooledDataSource();
+    DatabaseSchemaManager.createOrUpdateSchema(dataSource);
+
+    final Map<String, Object> hibernateProperties =
+        ImmutableMap.of(org.hibernate.cfg.AvailableSettings.DATASOURCE, dataSource);
+
+    entityManager =
+        Persistence.createEntityManagerFactory(PERSISTENCE_UNIT_NAME, hibernateProperties)
+            .createEntityManager();
+  }
+
+  public void seedData(Collection<?> entities) {
+    Iterator<?> i = entities.iterator();
+
+    while (i.hasNext()) {
+      doTransaction(em -> em.persist(i.next()));
+    }
```

Comment:
Should use a for loop here rather than an iterator:

````
    for (Object entity : entities) {
        doTransaction(em -> em.persist(entity));
    }
````

Reply:
Should this really be one transaction per object?